certified-connectors/HoneywellForge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,432 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Honeywell Forge",
+    "description": "Honeywell Forge is an Enterprise Performance Management Software-as-a-Service. It enables autonomous control of devices and codifies domain knowledge of assets in a contextual way. Use Honeywell Forge Connector to subscribe to Honeywell Forge to receive service cases and to close existing service cases.",
+    "version": "1.0",
+    "contact": {
+      "name": "Honeywell Support",
+      "url": "https://www.honeywell.com/us/en/contact/support"
```

Comment:
@ashishdotme Can you please add `"email"` property as well.

Documentation - [Add metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4b-product-or-end-service-metadata)